TestAssets/TestProjects/LightupLib/project.json

```diff
@@ -0,0 +1,386 @@
+{
+
+  "buildOptions": {
+    "emitEntryPoint": false
+  },
+  "dependencies": {},
+  "frameworks": {
+    "netcoreapp2.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "2.0.0-beta-*"
+        },
+        "Newtonsoft.Json": "9.0.1-beta1",
+        "Microsoft.CodeAnalysis.Analyzers": {
+          "version": "1.1.0",
+          "exclude": "all"
+        },
+        "Microsoft.CSharp": {
```

Comment:
Yes, without them, I have been running into failures in bringing the dependency of the lib.